Makefile

```diff
@@ -1,6 +1,9 @@
 .PHONY: build lint test clean run_image build_image download-test-vectors clean-vectors \
 	setup-hive test-pattern-default run-hive run-hive-debug clean-hive-logs
 
+default: stop-localnet-silent ethereum-package checkout-ethereum-package localnet
```

Comment:
I would use `default` like a help command

Reply:
Done! Now the default target lists the available targets.